pkg/plugins/golang/v3/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -80,7 +80,7 @@ import (
 
 // {{ .Resource.Kind }}Reconciler reconciles a {{ .Resource.Kind }} object
 type {{ .Resource.Kind }}Reconciler struct {
-	client.Client
+	Client client.Client
```

Comment:
we should not change this plugin deprecated

Reply:
We can only apply changes for kustomize/v2 and go/v4

Reply:
Indeed it brokes the v3 one: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kubebuilder/3855/pull-kubebuilder-test/1777774950044667904